src/Microsoft.AspNetCore.Mvc.Razor/TagHelpers/ITagHelperComponentManager.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+
+namespace Microsoft.AspNetCore.Mvc.Razor.TagHelpers
+{
+    /// <summary>
+    /// An implementation of this interface provides the collection of <see cref="ITagHelperComponent"/>s
+    /// that will be used by the <see cref="TagHelperComponentTagHelper"/>.
```

Comment:
Minor but `<see cref="TagHelperComponentTagHelper"/>s` is better than `the <see cref="TagHelperComponentTagHelper"/>` because there isn't just one `<see cref="TagHelperComponentTagHelper"/>`.